ballerina-shell/modules/shell-rt/build.gradle

```diff
@@ -16,15 +16,17 @@
  * under the License.
  */
 
-apply from: "$rootDir/gradle/javaProject.gradle"
-apply from: "$rootDir/gradle/ballerinaLangLibLoad.gradle"
+plugins {
+    id 'javaProject'
+    id 'ballerinaLangLibLoad'
+}
 
 description = 'Ballerina - Ballerina Shell Runtime Dependencies'
 
 group = 'io.ballerina'
 
 dependencies {
-    testImplementation('org.testng:testng')
+    testImplementation(libs.testng)
```

Comment:
```suggestion
    testImplementation libs.testng
```

Reply:
@Shadow-Devil We can have a common practise to use brackets or no brackets. My suggestion is to avoid using brackets